### PR TITLE
Support namespaced keywords

### PR DESCRIPTION
**Pull request by dignati**
An alternative would be to let the user provide functions to read and write keywords.

### CODE REVIEW

File: src/rethinkdb/utils.cljc [hunk 1]
```diff
@@ -30,3 +30,7 @@
 
 (defn snake-case [s]
   (string/replace (name s) \- \_))
+
+(defn keyword->str [k]
+  (-> (str k)
```
Comment (danielcompton) [inline src/rethinkdb/utils.cljc]: You can use `name` here instead

Comment (dignati) [inline src/rethinkdb/utils.cljc]: name removes the namespace from the keyword. This is exactly what I want to prevent.

Comment (danielcompton) [inline src/rethinkdb/utils.cljc]: yeah, I realised this as I went to sleep last night :)

**Comment by apa512** [comment]
Added.

**Pull request closed without merging**